build/tools/templates.go

```diff
@@ -354,3 +359,23 @@ func writeBackendClientConfigFile(config *Config, outputDir string) error {
 	_, err = io.Copy(out, in)
 	return err
 }
+
+func writeBackendExecScript(config *Config, outputDir string) error {
+	out, err := os.OpenFile(filepath.Join(outputDir, "backend/exec.sh"), os.O_CREATE|os.O_WRONLY, 0777)
```

Comment:
Just thinking, is it necessary to open the output file if the `config.Backend.ExecScript` is empty?

Reply:
So it looks like due to the way these backend files are renamed during [dh-exec-install](https://github.com/trezor/blockbook/blob/1f4398aa99c17b340744d75cd677e7e4260ca61d/build/templates/backend/debian/install) it is expected the files exist. I tried a couple different attempts at conditional rename without any luck. Let me know if you have an idea, otherwise going to leave it in line with existing conf files

Reply:
Well, you could make https://github.com/trezor/blockbook/blob/1f4398aa99c17b340744d75cd677e7e4260ca61d/build/templates/backend/debian/install#L6 conditional :) I would actually prefer it as I do not want to have unnecessary files around if the exec script is not used.

Reply:
To be more specific, include the line only if the exec script exists.

Reply:
I was attempting to make conditional with
```
[ -f exec.sh ] && exec.sh => {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/{{.Coin.Alias}}_exec.sh
```
but it didn't seem happy with bash evaluation in the dh-exec script. I will fiddle with it a bit more

Reply:
Wha I mean is that the file `build/templates/backend/debian/install` is a go template - you could put there something like
```
{{if .Backend.ExecScript}}exec.sh => {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/{{.Coin.Alias}}_exec.sh{{end}}
```

Reply:
doh :man_facepalming: yeah that totally works haha